rubix-common/src/main/java/com/qubole/rubix/common/metrics/BookKeeperMetrics.java

```diff
@@ -218,15 +218,17 @@ public String getMetricName()
   }
 
   /**
-   * Enum for metrics relating to daemon & service liveness.
+   * Enum for metrics relating to daemon & service health.
    */
-  public enum LivenessMetric
+  public enum HealthMetric
   {
-    METRIC_BOOKKEEPER_LIVE_WORKER_GAUGE("rubix.bookkeeper.live_workers.gauge");
+    LIVE_WORKER_GAUGE("rubix.bookkeeper.live_workers.gauge"),
+    CACHING_VALIDATED_WORKER_GAUGE("rubix.bookkeeper.caching_validated_workers.gauge"),
```

Comment:
Can you please rename the metric with new convention ?
Ex: Instead of rubix.bookkeeper.caching_validated_workers.**gauge** , it should be
rubix.bookkeeper.**gauge**.caching_validated_workers